java/F3DJavaBindings.cxx

```diff
@@ -138,17 +138,17 @@ extern "C"
 
   JNIEXPORT void JAVA_BIND(Camera, azimuth)(JNIEnv* env, jobject self, jdouble angle)
   {
-    GetEngine(env, self)->getWindow().getCamera().azimuth(angle);
+    GetEngine(env, self)->getWindow().getCamera().addAzimuth(angle);
```

Comment:
Please change the api name in the JAVA_BIND macro

Reply:
Changed.